invenio_oauthclient/handlers/base.py

```diff
@@ -58,7 +58,9 @@ def base_authorized_signup_handler(resp, remote, *args, **kwargs):
     """
     # Remove any previously stored auto register session key
     session.pop(token_session_key(remote.name) + "_autoregister", None)
-
+    # We set the remote in the session to be aware of which one is being used and, on log out redirect to
+    # the correct URL set in the OAUTHCLIENT_REMOTE_APPS for each remote
+    session["remote_name"] = remote.name
```

Comment:
does it make sense to namespace this, with a prefix?
See for example `current_app.config["OAUTHCLIENT_SESSION_KEY_PREFIX"]`, I don't know if it makes much sense.

Reply:
Agree on this one... Also, what happens e.g. when you connect multiple OAuth providers to your account? Would it be only the last one that ends up in the session variable? Would it make sense to keep a list/set of remotes like so:

```python
session.setdefault("remote_names", set())  # not sure if set is supported
session["remote_name"].add(remote.name)
```

Then you could in logout check which of the remotes has a `logout_url` specified and again make an arbitrary choice, e.g. the first one...

This sounds a bit overcomplicated, and I'm not sure if it makes perfect sense though.

Reply:
Prefixed as requested.
I am not sure if multiple accounts connected affects this, the remote_name is set on loging and although your account might be connected to multiple OAuth providers you always log in with only one of the, or am I wrong?

However, if I am wrong, looks to me that could be done in another issue as an improvement (?)